Doc/library/winreg.rst

```diff
@@ -87,7 +87,7 @@ This module offers the following functions:
       See :ref:`above <exception-changed>`.
 
 
-.. function:: CreateKeyEx(key, sub_key, reserved=0, access=KEY_WRITE)
+.. function:: CreateKeyEx(key, sub_key, reserved=0, access=KEY_WRITE, options=0, create_only=False)
```

Comment:
I think it is worth to make all new parameters keyword-only. Their order differs from the order in `RegCreateKeyExW()`, so passing them as positional arguments will be errorprone.

We can also deprecate passing `reserved` (which should not be here at first place) and `access` by keyword, but this is a different issue.